var/spack/repos/builtin/packages/py-statsmodels/package.py

```diff
@@ -23,36 +26,50 @@ class PyStatsmodels(PythonPackage):
     version("0.10.2", sha256="9cd2194c6642a8754e85f9a6e6912cdf996bebf6ff715d3cc67f65dadfd37cc9")
     version("0.10.1", sha256="320659a80f916c2edf9dfbe83512d9004bb562b72eedb7d9374562038697fa10")
 
-    depends_on("python@2.7:2.8,3.4:", when="@0.10.1:", type=("build", "link", "run"))
-    depends_on("python@3.6:", when="@0.12.1:", type=("build", "link", "run"))
+    # limits according to wheels available on PyPI
```

Comment:
Not sure if we want this or not, up to you

Reply:
I tested for example `py-statsmodels@0.13.2` builds fine with `python@3.11` contradicting these limitations, so I removed them.